catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/ws/SecureWebSocketServlet.java

```diff
@@ -44,20 +45,33 @@ public void destroy() {
     executor.shutdown();
   }
 
+  /*
+   Pass the TokenHolder into the WebSocket, in order to know when the user has logged out. Can't
+   pass the Session because Jetty won't let anything check session attributes unless there's a
+   request (excluding WebSocket requests) being actively fulfilled for that session.
```

Comment:
Wasn't sure if this comment block was overkill, but it's not a very obvious nuance.